tests/test_users.py

```diff
@@ -88,14 +88,20 @@ def test_remove_groups(self):
             self.assertEqual(len(user.groups), 2)
 
     def test_set_policy(self):
-        name = "test_sete_policy@test.com"
+        name = "test_set_policy@test.com"
         user = User(self.directory, name)
         with self.subTest("The initial user policy is default_policy, when the user is first created"):
-            self.assertEqual(user.lookup_policies(), [self.default_policy])
+            self.assertEqual(user.statement, self.default_policy)
```

Comment:
Should `user.lookup_policies()` still be tested for consistency?

Reply:
This is tested else where.

Reply:
added test anyways